schema/crdb/dbinit.sql

```diff
@@ -213,7 +213,7 @@ CREATE INDEX IF NOT EXISTS lookup_sled_by_policy_and_state ON omicron.public.sle
 );
 
 CREATE TYPE IF NOT EXISTS omicron.public.sled_resource_kind AS ENUM (
-    -- omicron.public.instance
+    -- omicron.public.vmm ; this is called "instance" for historical reasons.
```

Comment:
This tripped me up. This is *not* the UUID of an instance, it's the UUID of a *propolis VMM* that has been allocated. These are very subtly different objects.